README.md

```diff
@@ -4,24 +4,24 @@
 
 Here you can find course content and homework for the JavaScript 1,2 and 3 modules
 
-|Week|Topic|Read|Homework|Review|
-|----|-----|----|--------|------|
-|0.|Preparation for your first JavaScript session|[Pre-reading](/Week0/README.md)|-|-|
-|1.|Git Session with Unmesh|[Reading Week 1](/Week1/README.md) | [Homework Week 1](/Week1/MAKEME.md)|[Review](/Week1/REVIEW.md)|
-|2.|• Intro JavaScript (What is it, where can you use it for)<br>• Variables [var, let, const]<br>• Basic Data types [Strings, Numbers, Arrays, Booleans]<br>• Operators <br>• Naming conventions|[Reading Week 2](/Week2/README.md)|[Homework Week 2](/Week2/MAKEME.md)|[Review](/master/Week2/REVIEW.md)|
-|3.|• Git work flow :smiling_imp:<br>• Advanced data types [Objects] <br>• Conditions <br>• Statements vs Expressions<br> • Loops (for/while)<br>• Functions <br>• Scope|[Reading Week 3](/Week3/README.md)|[Homework Week 3](/Week3/MAKEME.md)|[Review](/Week3/REVIEW.md)|
-|4.|• Capturing user input <br>• Events • Basic DOM manipulations[img src, innerHTML] <br>• Code debugging using the browser <br>• Code commenting <br>• Structuring code files |[Reading Week 4](/Week4/README.md)|[Homework Week 4](/Week4/MAKEME.md)|Review|
-|5.|• Functions + JSON/Arrays<br>• Array Manipulations<br>• JSON<br>• Map and filter<br>• Arrow functions |[Reading Week 5](/Week5/README.md)|[Homework Week 5](/Week5/MAKEME.md)|[Review](/Week5/REVIEW.MD)|
-|6.|• Closures <br>• Callbacks|[Reading Week 6](/Week6/README.md)|[Homework Week 6](/Week6/MAKEME.md)|[Review](/Week6/REVIEW.md)|
-|7.|• Object Oriented Programming  <br>• Code flow (order of execution) <br>• Async VS Sync|[Reading Week 7](/Week7/README.md)|[Homework Week 7](/Week7/MAKEME.md)|[Review](/Week7/REVIEW.md)|
-|8.|• Structure for a basic SPA <br>• XHTTP Requests <br>• API calls|[Reading Week 8](/Week8/README.md)|[Homework Week 8](/Week8/MAKEME.md)|Review|
-|9.|• (re)writing data structures (in JSON) <br>• Promises|[Reading Week 9](/Week9/README.md)|[Homework Week 9](/Week9/MAKEME.md)|[Review](/Week9/REVIEW.md)|
+|Week|Topic|Read|Homework|
+|----|-----|----|--------|
+|0.|Preparation for your first JavaScript session|[Pre-reading](/Week0/README.md)|-|
+|1.|Git Session with Unmesh|[Reading Week 1](/Week1/README.md) | [Homework Week 1](/Week1/MAKEME.md)|
+|2.|• Intro JavaScript (What is it, where can you use it for)<br>• [Variables (var, let, const)](fundamentals/variables.md)<br>• [Basic Data types (strings, numbers, arrays, booleans)](fundamentals/values.md)<br>• [Operators](fundamentals/operators.md)<br>• [Naming conventions](fundamentals/naming_conventions.md)|[Reading Week 2](/Week2/README.md)|[Homework Week 2](/Week2/MAKEME.md)|
```

Comment:
Why did you change (strings, numbers, arrays, booleans) to lowercase?

Reply:
String, Number and Boolean refer to the constructor functions. The lowercase versions refer to the primitive values. But if you prefer to stick to uppercase, that's fine with me too.